styleguide/placeholder.scss

```diff
@@ -74,3 +83,47 @@
     transition: 50ms fill ease-out;
   }
 }
+
+.kiln-placeholder .placeholder-add-component {
```

Comment:
Yes, I guess I was suggesting a refactor of the organization of the file, but not necessary for this PR.